components/schemes/Typography/tests/unit/MDCTypographySchemeTests.m

```diff
@@ -149,4 +149,22 @@ - (void)testTypographyScheme201902ReturnsDefaultSizedFonts {
       mdc_isSimplyEqual:[scheme201902.overline mdc_scaledFontAtDefaultSize]]);
 }
 
+- (void)testTypographyScheme201804DisableUseCurrentContentSizeCategoryWhenAppliedByDefault {
```

Comment:
suggestion: add a test to make sure mdc_adjustsFontForContentSizeCategory is imitating the useCurrentContentSizeCategoryWhenApplied behavior.

Reply:
Good idea! Added tests.